crates/solidity/testing/snapshots/cst_output/InterfaceDefinition/sample_counter/generated/0.4.11.yml

```diff
@@ -15,44 +15,46 @@ Tree:
   - InterfaceDefinition (Rule): # 0..172 "interface ICounter {\n    // returns the current co..."
       - Interface (Token): "interface" # 0..9
       - Identifier (Token): "ICounter" # 10..18
-      - Group: # 19..172 "{\n    // returns the current count\n    function co..."
+      - _DELIMITEDBY (Rule): # 18..172 " {\n    // returns the current count\n    function c..."
           - OpenBrace (Token): "{" # 19..20
-          - Group: # 25..170 "// returns the current count\n    function count() ..."
-              - ContractBodyElement (Rule): # 25..104 "// returns the current count\n    function count() ..."
-                  - FunctionDefinition (Rule): # 25..104 "// returns the current count\n    function count() ..."
+          - _REPEATED (Rule): # 21..171 "    // returns the current count\n    function coun..."
+              - ContractBodyElement (Rule): # 21..106 "    // returns the current count\n    function coun..."
+                  - FunctionDefinition (Rule): # 21..106 "    // returns the current count\n    function coun..."
                       - Function (Token): # 25..66 "// returns the current count\n    function"
                           - SingleLineComment (Trivia): "// returns the current count" # 25..53
                           - Contents: "function" # 58..66
                       - Identifier (Token): "count" # 67..72
                       - ParameterList (Rule): # 72..74 "()"
-                          - OpenParen (Token): "(" # 72..73
-                          - CloseParen (Token): ")" # 73..74
-                      - Group: # 75..88 "external view"
+                          - _DELIMITEDBY (Rule): # 72..74 "()"
```

Comment:
Love this! That destinction `_DELIMITEDBY`, `_REPEATED` makes it much easier to understand/fix the grammar